.nightly

```diff
@@ -1 +1,2 @@
 repo: develop
+branch: release-25.0.0
```

Comment:
Instead of `branch:` which is kind of ambiguous (what branch? this branch?) should this maybe be `parent_branch:` or `root_branch:` or something?

Reply:
I'll update it to be `parent_branch`